### PR TITLE
Allow to dynamically override LEAFLET_CONFIG, minor fixes, czech locale

### PR DESCRIPTION
**Pull request by PetrDlouhy**
I needed to set DEFAULT_CENTER and DEFAULT_ZOOM depending on the user city location. Here are changes which allow to set LEAFLET_CONFIG overrides in LeafletWidget init parameter. Documentation of this is also part of the PR.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -501,6 +501,29 @@ Advanced usage
   If empty values is provided, then map initialization is immediate.
   And with a wrong value, the map is never initialized. :)
 
+* ``settings_overrides``: Map with overrides to the default LEAFLET_CONFIG settings.
+  (Default: {}).
```
Comment (leplatrem) [inline README.rst]: Dict to override the default...

**Comment by leplatrem** [comment]
Nice contribution ! Thanks !

Would you mind adding a small test case please ?

**Comment by PetrDlouhy** [comment]
OK, I will try.

**Comment by PetrDlouhy** [comment]
Now it is with the test.

**Comment by leplatrem** [comment]
Thanks a lot !

**Pull request merged**